coding/javascript.md

```diff
@@ -236,7 +237,8 @@ We recommend new projects use the [hooks API](https://reactjs.org/docs/hooks-int
 All projects should generally use the following:
 
 - Style framework - [Vanilla](https://github.com/vanilla-framework/vanilla-framework)
-- Component testing - [Enzyme](https://github.com/airbnb/enzyme)
+- Component testing(New) - [React Testing Library](https://github.com/testing-library/react-testing-library)
```

Comment:
I was actually going to be introducing it in the current branch that I'm working on for the Juju Dashboard :)

Reply:
As you know by now we're already using this in various places (e.g. ubuntu.com repo). About time we include it in the docs👍

Reply:
I think this might have been a little hasty to merge as we're still discussing how best to use RTL while still being confident that we're testing effectively.

Reply:
It's just a link to the testing library, we can always add recommendations in a new PR?